SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -937,6 +937,23 @@ - (OWSMessageSend *)getSessionRestoreMessageForHexEncodedPublicKey:(NSString *)h
     return [[OWSMessageSend alloc] initWithMessage:message thread:thread recipient:recipient senderCertificate:senderCertificate udAccess:theirUDAccess localNumber:userHexEncodedPublicKey success:^{ } failure:^(NSError *error) { }];
 }
 
+- (LKFriendRequestMessage *)getMultiDeviceFriendRequestMessageForHexEncodedPublicKey:(NSString *)hexEncodedPublicKey inThread:(TSContactThread *)thread transaction:(YapDatabaseReadWriteTransaction *)transaction
+{
+    thread = [TSContactThread getOrCreateThreadWithContactId:hexEncodedPublicKey transaction:transaction];
```

Comment:
This doesn't use the thread that's passed in?

Reply:
Oh boy, this line of code is duplicated. This `thread` is exactly the thread that's passed in...